editor/src/components/canvas/canvas-strategies/strategies/drag-to-move-metastrategy.tsx

```diff
@@ -0,0 +1,108 @@
+import { ImmediateParentBounds } from '../../controls/parent-bounds'
+import { ImmediateParentOutlines } from '../../controls/parent-outlines'
+import {
+  DragOutlineControl,
+  dragTargetsElementPaths,
+} from '../../controls/select-mode/drag-outline-control'
+import { CanvasStrategyFactory, MetaCanvasStrategy } from '../canvas-strategies'
+import {
+  CustomStrategyState,
+  getTargetPathsFromInteractionTarget,
+  InteractionCanvasState,
+  CanvasStrategy,
+  emptyStrategyApplicationResult,
+  controlWithProps,
+} from '../canvas-strategy-types'
+import { InteractionSession } from '../interaction-state'
+import { absoluteMoveStrategy } from './absolute-move-strategy'
+import { flexReorderStrategy } from './flex-reorder-strategy'
+import { flowReorderStrategy } from './flow-reorder-strategy'
+import { lookForApplicableParentStrategy } from './look-for-applicable-parent-strategy'
+import { relativeMoveStrategy } from './relative-move-strategy'
+import { reparentMetaStrategy } from './reparent-metastrategy'
+import { getDragTargets } from './shared-move-strategies-helpers'
+
+const baseMoveStrategyFactories: Array<CanvasStrategyFactory> = [
+  absoluteMoveStrategy,
+  flexReorderStrategy,
+  flowReorderStrategy,
+  relativeMoveStrategy,
+]
+
+export const dragToMoveMetaStrategy: MetaCanvasStrategy = (
+  canvasState: InteractionCanvasState,
+  interactionSession: InteractionSession | null,
+  customStrategyState: CustomStrategyState,
+): Array<CanvasStrategy> => {
+  const selectedElements = getDragTargets(
+    getTargetPathsFromInteractionTarget(canvasState.interactionTarget),
+  )
+
+  if (
+    selectedElements.length === 0 ||
+    interactionSession == null ||
+    interactionSession.activeControl.type !== 'BOUNDING_AREA' ||
+    interactionSession.interactionData.type !== 'DRAG' ||
+    interactionSession.interactionData.modifiers.alt
+  ) {
+    return []
+  }
+
+  const ancestorStrategies = lookForApplicableParentStrategy(
+    canvasState,
+    interactionSession,
+    customStrategyState,
+  )
+  const reparentStrategies = reparentMetaStrategy(
+    canvasState,
+    interactionSession,
+    customStrategyState,
+  )
+  const dragStrategies = mapDropNulls(
+    (factory) => factory(canvasState, interactionSession, customStrategyState),
+    baseMoveStrategyFactories,
+  )
+  const foundStrategies = [...ancestorStrategies, ...reparentStrategies, ...dragStrategies]
+  if (foundStrategies.length > 0) {
+    return foundStrategies
+  } else {
+    return [doNothingStrategy(canvasState, interactionSession, customStrategyState)]
+  }
+}
+
+export function doNothingStrategy(
+  canvasState: InteractionCanvasState,
+  interactionSession: InteractionSession | null,
+  customStrategyState: CustomStrategyState,
+): CanvasStrategy {
+  const selectedElements = getTargetPathsFromInteractionTarget(canvasState.interactionTarget)
+  return {
+    id: 'DO_NOTHING',
+    name: 'Placeholder',
```

Comment:
I think we should find a better name..... Placeholder is pretty cryptic.

Reply:
oh yes, maybe 'Move' would fit better :)